### PR TITLE
Correct Grammatical Errors

### PR DESCRIPTION
**Pull request by Tloru**
Corrected the following issues:
- Fixed possession of some nouns.
- Changed improper uses of 'which' → 'that'
- Cleaned up/clarified some prose
- & Corrected common grammatical errors

😄

### CODE REVIEW

File: src/content/site/rules.md [hunk 1]
```diff
@@ -139,13 +139,13 @@ These people will try to provide some answers on their own time for questions in
...
+Self-explanatory - _ChatMod/Moderator/Senior Moderator/Admin/Owner_ - These people ensure that the usage of this environment reflects the guidelines as illustrated in the rules. Staff is occasionally handpicked, and sometimes there is an application process. There is a trial period for all new staff of 2-4 weeks (Sometimes more, sometimes less) which comes with a trial role, however, they are treated as full staff members during this time. Please respect all staff equally; if you have an issue with a specific staff member, talk to an administrator.
 
 [Table of Contents](#table-of-contents)
 
 # Nickname Policy
 
-On TPH, we enforce a nickname policy - This means that if your nickname is deemed too NSFW (not safe for work), or hard to read/tag (it uses weird characters, it's blank or it contains numbers) we can and will forcibly nickname you to something else.
+On TPH, we enforce a nickname policy - This means that if your nickname is deemed to NSFW (not safe for work), or hard to read/tag (it uses weird characters, it's blank or it contains numbers) we can and will forcibly nickname you to something else.
```
Comment (veksen) [inline src/content/site/rules.md]: too NSFW seems right, no?

File: src/content/site/rules.md [hunk 2]
```diff
@@ -123,14 +123,14 @@ Try not to, honestly, if you are looking to prevent a specific kind of attack fe
...
+_Community-Member, Active Contributor, Chatterbox, Super-Talky-Guy, Bepis-Drinker, Conke-Commander, and Patriotic-Procrastinator_
 
-These are the activity roles of the server. Different tiers will grant different perks, such unlocking hidden channels, voice access, and nicknaming privileges. These roles are assigned automatically as you participate in the server, so don't ask for them.
+These are the activity roles of the server. Different tiers will grant different perks, such as unlocking hidden channels, voice access, and nicknaming privileges. These roles are assigned automatically as you participate in the server, so don't ask for them.
 
 **Wizard**<br />
-This is for helpful members who have shown exceptional aptitude or expertise in some field. This is given out on the discretion of Elliott - though other staff members may suggest someone to get it to Elliott. Less than 0.001% of users get this role.
+This is for helpful members who have shown exceptional aptitude or expertise in some field. This is given out at the discretion of Elliott - though other staff members may suggest for Elliot to give it to someone. Less than 0.001% of users get this role.
```
Comment (veksen) [inline src/content/site/rules.md]: As we're here, can we mention Elliot and Fox?

File: src/content/site/rules.md (hunk 1 above)
Comment (Tloru) [reply src/content/site/rules.md]: You're right, Will change.

File: src/content/site/rules.md (hunk 2 above)
Comment (Tloru) [reply src/content/site/rules.md]: What about them?

**Comment by Tloru** [comment]
While we're at it, it might help to review other files too.

File: src/content/site/rules.md (hunk 2 above)
Comment (veksen) [reply src/content/site/rules.md]: I mean, Elliott and Fox co-own. I think this text is legacy. You editing this bit made it obvious.

**Comment by veksen** [comment]
thank you for your contribution! <3

**Pull request merged**